src/main/java/org/skriptlang/skript/bukkit/loottables/elements/expressions/ExprLootContextEntity.java

```diff
@@ -0,0 +1,91 @@
+package org.skriptlang.skript.bukkit.loottables.elements.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.bukkit.loot.LootContext;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.loottables.LootContextCreateEvent;
+import org.skriptlang.skript.bukkit.loottables.LootContextWrapper;
+
+@Name("Loot Entity of Loot Context")
+@Description("Returns the loot context entity of a loot context.")
+@Examples({"set {_entity} to loot context entity of {_context}",
+	"",
+	"set {_context} to a new loot context at player:",
+		"\tset loot context luck value to 10",
+		"\tset loot context killer to player",
+		"\tset loot context entity to last spawned pig"
+})
+@Since("INSERT VERSION")
+public class ExprLootContextEntity extends PropertyExpression<LootContext, Entity> {
```

Comment:
Why not use a simple property expression? you can use `registerDefault` that way